### ISSUE TITLE
"headless" support (running plugin without UI)

### ISSUE THREAD

**Issue by drtebi** [sadko4u/lsp-plugins#110]
Hello,

maybe I missed something, but as far as I can tell, it is not possible to run a plugin without its UI. I would greatly appreciate it if that would be made possible.

The story behind this is quite simple: I am using EQ filters created in REW, which I then import into the lsp-plugins-parametric-equalizer-x16-stereo, to tweak the frequency response of a pair of full-range drivers I have. The results are stunning, to say the least. 
Ideally however, I would want to run a headless Raspberry-Pi with MPD, routing (via JACK) its output to the EQ and then to the system outputs. No UI would be necessary, since the EQ settings would not need to be changed again anyway.

I am aware that there are other solutions for this idea, like BruteFir. But it is more tricky to set up, and I also like the lsp-plugin solution better, because I can easily tweak the response further in the UI, then save the configuration and just load it on start-up.

Maybe it's too much to ask for, but I just had to ask anyway :)

**Comment by sadko4u** [comment]
Hello!
Thank you for your response.
Yes, I also thought that making UI optional for the JACK version of plugins could be a good solution.
Anyway, that feature won't be earlier than in 1.2.0.

**Comment by drtebi** [comment]
That sounds great. This could really mean an easy setup with just the raspberry-pi, an amplifier, and a pair of full-range speakers :)

I will happily to wait for version 1.2.0.

**Comment by falkTX** [comment]
FYI you can already build lv2 DSP only, skipping UI.
I am using these as makefile arguments:
```
	BUILD_MODULES=lv2 \
	BUILD_PROFILE=aarch64 \
	BUILD_R3D_BACKENDS= \
	JACK_HEADERS= \
	JACK_LIBS= \
	LV2_UI=0 \
	PREFIX=/usr \
	XLIB_HEADERS= \
	XLIB_LIBS=
```

**Comment by sadko4u** [comment]
@falkTX  this is for LV2, not JACK. @drtebi asked about JACK as far as I remember.

**Comment by sadko4u** [comment]
Available in 1.2.0.
